packages/next/src/client/script.tsx

```diff
@@ -68,7 +68,7 @@ const insertStylesheets = (stylesheets: string[]) => {
   }
 }
 
-const loadScript = (props: ScriptProps): void => {
+export const loadScript = (props: ScriptProps): void => {
```

Comment:
I think we might not want to expose new API from `next/script` to user land, are we able to use the component itself instead?

Reply:
Interesting. I will sync up with you to see how that would work. We had decided against using the component in our previous chats, but want to confirm what you mean